homeassistant/components/tessie/strings.json

```diff
@@ -0,0 +1,92 @@
+{
+  "config": {
+    "error": {
+      "invalid_access_token": "[%key:common::config_flow::error::invalid_access_token%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "description": "Enter your access token from [my.tessie.com/settings/api](https://my.tessie.com/settings/api)."
```

Comment:
URLs should be placeholders in the description to not be translated. Search for `description_placeholders` for examples.